### ISSUE TITLE
[Bug] error: package android.support.v4.graphics does not exist 

### ISSUE THREAD

**Issue by nighttiger1990** [react-native-community/art#51]
# Bug

## Environment info

React native info output:

```bash
info Fetching system and libraries information...
System:
    OS: macOS Mojave 10.14.5
    CPU: (4) x64 Intel(R) Core(TM) i5-3210M CPU @ 2.50GHz
    Memory: 61.13 MB / 8.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.15.3 - /usr/local/bin/node
    Yarn: 1.15.2 - /usr/local/bin/yarn
    npm: 6.4.1 - /usr/local/bin/npm
  SDKs:
    iOS SDK:
      Platforms: iOS 13.2, DriverKit 19.0, macOS 10.15, tvOS 13.2, watchOS 6.1
    Android SDK:
      API Levels: 27, 28
      Build Tools: 27.0.3, 28.0.3
      System Images: android-28 | Google Play Intel x86 Atom
  IDEs:
    Android Studio: 3.3 AI-182.5107.16.33.5314842
    Xcode: 11.3/11C29 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.9.0 => 16.9.0 
    react-native: 0.61.5 => 0.61.5 
  npmGlobalPackages:
    react-native-cli: 2.0.1
```

Library version: x.x.x

## Steps To Reproduce

1. `cd android`
2. `gradlew assembleDebug`
3. got error message

```
error: package android.support.v4.graphics does not exist import android.support.v4.graphics.ColorUtils;
```

**Comment by nighttiger1990** [comment]
after googling, i think this relate to AndroidX support. Then i run
`npx jetify` and everything work like a charm.

**Comment by vance-liu** [comment]
Break change, please add AndroidX support.
change
`
android.support.v4.graphics.ColorUtils
`
to
`
androidx.core.graphics.ColorUtils
`

**Comment by henrymoulton** [comment]
Can we reopen this issue if this repository plans on supporting AndroidX?

**Comment by enigmablue** [comment]
Same error. It occurs on android studio
But CLI seems to be OK i think because of jetify.

But i keep getting this on appcenter occasionally. Any idea?

**Comment by a-koka** [comment]
Same issue, but getting error in command line. `npx jetify` fixed the problem though. Would be nice to understand why.

**Comment by nighttiger1990** [comment]
> Can we reopen this issue if this repository plans on supporting AndroidX?

I re-open this. Hope this support AndroidX soon

**Comment by Esemesek** [comment]
It should be fixed in `1.2.0` release.

**Comment by mangBert** [comment]
@vance-liu  works like a charm :)
